Modules/Models/Configuration.swift

```diff
@@ -80,11 +80,17 @@ public struct Configuration: Codable {
   /// pass before a foreground session should force a new one.
   public let maximumExposureDetectionWaitingTime: TimeInterval
 
-  /// The url of the privacy policy
-  public let privacyPolicyURL: URL
+  /// The url of the privacy notice
+  /// - note: this dictionary uses string as a key because only strings and ints
+  /// are really considered as dictionaries by Codable
+  /// https://bugs.swift.org/browse/SR-7788
```

Comment:
[i] We could probably work around this with property wrappers instead of repeating this comment over and over, but it's very low priority